misc/mdns/src/behaviour.rs

```diff
@@ -90,7 +90,7 @@ impl<TSubstream> Mdns<TSubstream> {
     /// Builds a new `Mdns` behaviour.
     pub async fn new() -> io::Result<Mdns<TSubstream>> {
```

Comment:
```suggestion
    pub fn new() -> io::Result<Mdns<TSubstream>> {
```
?

Reply:
Right. That makes a lot of sense. Thanks.

Reply:
Right. That makes a lot of sense. Thanks.